src/core/components/info/info-base-path.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export default class InfoBasePath extends React.Component {
```

Comment:
Yeah - I guess I wasn't sure of the value of making stateless components as the `wrapRender` method will convert them to a class in any case. Will update anyway to be consistent

Reply:
No worries, good thinking on that front. In this case it'd be good to have SFCs since they're nicer to look at, and we can someday optimize `wrapRender` to preserve the pattern for performance.

Reply:
Done.